2.1/jessie/amd64/sdk/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM microsoft/dotnet-nightly:2.1.300-preview1-sdk-jessie
+
+# set up environment
+ENV ASPNETCORE_URLS http://+:80
+
+# Ensure packages required for standalone apps are pre-fetched
+RUN mkdir /tmp/warmup && cd /tmp/warmup \
```

Comment:
I find the mixture of multi-command lines  (e.g. && cd / && rm -rf /tmp/warmup) and single lines to be a little confusing.